### ISSUE TITLE
psnr value

### ISSUE THREAD

**Issue by rookiexiao123** [codeslake/RealTime_VDBLR#7]
May I ask if the indicators of PSNR in the paper are for the training set or the validation set?
